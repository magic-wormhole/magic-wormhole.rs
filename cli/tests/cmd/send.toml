bin.name = "wormhole-rs"
args = "send"
status.code = 2
