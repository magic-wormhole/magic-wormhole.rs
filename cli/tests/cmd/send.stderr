error: the following required arguments were not provided:
  <FILENAME|DIRNAME>...

Usage: wormhole-rs[EXE] send <FILENAME|DIRNAME>...

For more information, try '--help'.
